### PR TITLE
Slots accordion

### PR DESCRIPTION
**Pull request by quentinderoubaix**

### CODE REVIEW

File: core-bootstrap/src/components/accordion/accordion.ts [hunk 1]
```diff
@@ -26,26 +26,35 @@ interface AccordionExtraProps {
...
 	 * contains accordion header and accordion-item body container; the accordion header contains the accordion button
-	 * (that contains `slotItemHeader`), while the accordion-item body container contains the accordion body (that contains `slotItemBody`).
-	 * The itemTransition it applied on this element.
+	 * (that contains `header`), while the accordion-item body container contains the accordion body (that contains `children`).
+	 * The itemTransition is applied on this element.
 	 *
 	 * It is a prop of the accordion-item.
 	 */
-	slotItemStructure: SlotContent<AccordionItemContext>;
+	itemStructure: SlotContent<AccordionItemContext>;
```
Comment (andreascorti) [inline core-bootstrap/src/components/accordion/accordion.ts:34]: I thought this was replaced by `structure`. Do you also need `itemStructure` ?

File: core-bootstrap/src/components/accordion/accordion.ts [hunk 2]
```diff
@@ -59,12 +68,12 @@ export interface AccordionItemProps extends CoreItemProps, AccordionItemExtraPro
 export type AccordionItemWidget = Widget<AccordionItemProps, AccordionItemState, AccordionItemApi, AccordionItemActions, AccordionItemDirectives>;
 
 const defaultConfigExtraProps: AccordionExtraProps = {
-	slotItemStructure: undefined,
+	itemStructure: undefined,
```
Comment (andreascorti) [inline core-bootstrap/src/components/accordion/accordion.ts:71]: same as before

File: angular/bootstrap/src/components/accordion/accordion.component.ts [hunk 3]
```diff
@@ -135,7 +137,7 @@ export class AccordionItemDefaultSlotsComponent {
 export const accordionItemDefaultSlotItemStructure = new ComponentTemplate(AccordionItemDefaultSlotsComponent, 'structure');
```
Comment (andreascorti) [inline angular/bootstrap/src/components/accordion/accordion.component.ts:137]: Don't you want to change also variable names to reflect the correct name of the slot? `accordionItemDefaultSlotItemStructure` -> `accordionItemDefaultSlotStructure` ?

File: angular/bootstrap/src/components/accordion/accordion.component.ts [hunk 4]
```diff
@@ -315,15 +325,16 @@ export class AccordionDirective extends BaseWidgetDirective<AccordionWidget> {
...
 	 * contains accordion header and accordion-item body container; the accordion header contains the accordion button
-	 * (that contains `slotItemHeader`), while the accordion-item body container contains the accordion body (that contains `slotItemBody`).
-	 * The itemTransition it applied on this element.
+	 * (that contains `header`), while the accordion-item body container contains the accordion body (that contains `children`).
+	 * The itemTransition is applied on this element.
 	 *
 	 * It is a prop of the accordion-item.
 	 */
-	@Input('auSlotItemStructure') slotItemStructure: SlotContent<AccordionItemContext>;
+	@Input('auItemStructure') itemStructure: SlotContent<AccordionItemContext>;
```
Comment (andreascorti) [inline angular/bootstrap/src/components/accordion/accordion.component.ts:337]: Is it used?

**Review by andreascorti — COMMENTED** [review]
LGTM, some extra properties probably to polish.

**Review by andreascorti — APPROVED** [review]
turned out I didn't understand the override. LGTM

**Comment by quentinderoubaix** [comment]
thanks for the review @andreascorti !

**Pull request merged**